RadarSDK/RadarAPIClient.m

```diff
@@ -68,6 +71,8 @@ + (NSDictionary *)headersWithPublishableKey:(NSString *)publishableKey {
         @"X-Radar-Device-Type": [RadarUtils deviceType],
         @"X-Radar-SDK-Version": [RadarUtils sdkVersion],
         @"X-Radar-Mobile-Origin": [[NSBundle mainBundle] bundleIdentifier],
+        @"X-Radar-Network-Type": [RadarUtils networkTypeString],
+        @"X-Radar-App-Info": appInfoJSON
```

Comment:
First thought I was imagining these in the track payload but I suppose headers makes sense... could be useful for other requests as well.

Thinking out loud: is there any risk here? Nothing is occurring to me off the top of my head (payload size impact is small...)